web_common/web_common/templates/layout.html

```diff
@@ -8,6 +8,16 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="{{ base_path }}/common_static/css/main.css" />
     {% block head %}{% endblock %}
+    <script src="//cdnjs.cloudflare.com/ajax/libs/mousetrap/1.4.6/mousetrap.min.js"></script>
+    <script type="text/javascript">
+      function search() {
```

Comment:
I like my solution better than this one. PR forthcoming once I have a chance to dev deploy.